### PR TITLE
add recipe for libdrm

### PR DESCRIPTION
**Pull request by vchuravy**

### CODE REVIEW

File: L/libdrm/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    Linux(:i686, libc=:glibc),
+    Linux(:x86_64, libc=:glibc),
+    Linux(:aarch64, libc=:glibc),
+    Linux(:armv7l, libc=:glibc, call_abi=:eabihf),
+    Linux(:powerpc64le, libc=:glibc),
+    Linux(:i686, libc=:musl),
+    Linux(:x86_64, libc=:musl),
+    Linux(:aarch64, libc=:musl),
+    Linux(:armv7l, libc=:musl, call_abi=:eabihf)
+]
```
Comment (giordano) [inline L/libdrm/build_tarballs.jl:37]:
```suggestion
platforms = filter(Sys.islinux, supported_platform())
```

File: L/libdrm/bundled/patches/meson_amdgpu.patch [hunk 2]
```diff
@@ -0,0 +1,13 @@
+diff --git before/libdrm-2.4.103/amdgpu/meson.build after/libdrm-2.4.103/amdgpu/meson.build
```
Comment (giordano) [inline L/libdrm/bundled/patches/meson_amdgpu.patch:1]: It'd be good to upstream this

File: L/libdrm/build_tarballs.jl [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    atomic_patch -p1 ${f}
+done
+cd libdrm-*
+meson --cross-file=${MESON_TARGET_TOOLCHAIN} -Dudev=false -Dvalgrind=false build
+ninja -C build install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = filter(Sys.islinux, supported_platform())
```
Comment (giordano) [inline L/libdrm/build_tarballs.jl:27]: Meh
```suggestion
platforms = filter(Sys.islinux, supported_platforms())
```

File: L/libdrm/bundled/patches/meson_amdgpu.patch (hunk 2 above)
Comment (vchuravy) [reply L/libdrm/bundled/patches/meson_amdgpu.patch:1]: https://gitlab.freedesktop.org/mesa/drm/-/merge_requests/100

**Review by giordano — APPROVED** [review]

**Comment by vchuravy** [comment]
Thanks Mose!

File: L/libdrm/bundled/patches/meson_amdgpu.patch (hunk 2 above)
Comment (giordano) [reply L/libdrm/bundled/patches/meson_amdgpu.patch:1]: :100:

**Pull request merged**